cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -251,7 +304,7 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
         }
 
         p("Like other `SHOW` commands, the output can also be processed using `YIELD` / `WHERE` / `RETURN`. " +
-          "Additionally, just as with role privileges, it is also possible to show user privileges as revoke commands.")
+          "Additionally, just as with other privileges, it is also possible to show the commands for revoking the privileges.")
```

Comment:
```suggestion
          "Additionally, similar to other privileges, it is also possible to show the commands for revoking the privileges.")
```

Reply:
```suggestion
          "Additionally, similar to the other show privilege commands, it is also possible to show the commands for revoking the privileges.")
```

Reading your edit make me want to be clearer in general

Reply:
👍  Agreed!